operator/docs/operator/api.md

```diff
@@ -11,802 +10,5268 @@ menu:
 weight: 1000
 toc: true
 ---
+This Document contains the types introduced by the Loki Operator to be consumed by users.
+> This page is automatically generated with `gen-crd-api-reference-docs`.
+# loki.grafana.com/v1 { #loki-grafana-com-v1 }
+<div>
+<p>Package v1 contains API Schema definitions for the loki v1 API group</p>
+</div>
+Resource Types:
+<ul></ul>
```

Comment:
Should there be a list of resources here?

Reply:
No,  this is just a derp moment where I forgot to delete this